editors/emacs/Portfile

```diff
@@ -59,13 +62,20 @@ depends_lib-append     port:gmp \
                        path:lib/pkgconfig/gnutls.pc:gnutls \
                        port:jansson \
                        port:libxml2 \
-                       port:ncurses
+                       port:ncurses \
+                       port:sqlite3 \
+                       port:webp
 
 post-destroot {
     xinstall -d ${destroot}${prefix}/share/emacs/${version}/leim
     delete ${destroot}${prefix}/bin/ctags
     delete ${destroot}${prefix}/share/man/man1/ctags.1.gz
 
+    file copy ${filespath}/site-start.el \
+        ${destroot}${prefix}/share/emacs/site-lisp/
+    reinplace "s|__PREFIX__|${prefix}|g" \
+        ${destroot}${prefix}/share/emacs/site-lisp/site-start.el
```

Comment:
I think that it need to make clear that this line overwrite existed `site-start.el` from non `-app` version.

Reply:
Are you saying it's unclear that this is originally `site-start.el` and not `site-start-app.el`? Or are you saying it's unclear that that we are overwriting the original `site-start.el` provided by Emacs itself?

If the former, I feel like that's clear just by looking at the target of `reinplace`.

If the latter, Emacs does not provide a `site-start.el` so we are not overwriting it.

Reply:
The first one